api/v1/server/oas/transformers/worker.go

```diff
@@ -63,19 +67,27 @@ func ToWorker(worker *db.WorkerModel) *gen.Worker {
 	return res
 }
 
-func ToWorkerSqlc(worker *dbsqlc.Worker, stepCount *int64, slots *int) *gen.Worker {
+func ToWorkerSqlc(worker *dbsqlc.Worker, slots *int) *gen.Worker {
 
 	dispatcherId := uuid.MustParse(pgUUIDToStr(worker.DispatcherId))
 
 	maxRuns := int(worker.MaxRuns)
 
 	status := gen.ACTIVE
 
+	if worker.IsPaused {
+		status = gen.PAUSED
+	}
```

Comment:
I think we might need to move this below the next if, so that it shows paused as long as it is paused, even when it is offline

Reply:
i'm not sure, i think we want to surface inactive even if it is paused since a paused worker may go offline and never come back online